learning/tour-of-beam/frontend/README.md

```diff
@@ -17,47 +17,62 @@
      under the License.
  -->
 
- # Tour of Beam
- These are the main sources of the Tour of Beam website.
+# Tour of Beam
 
- # About
+These are the main sources of the Tour of Beam website.
+
+# About
 
 ## Getting started
+
 Running, debugging, and testing all require this first step that fetches
 dependencies and generates code:
 
 ```bash
 cd ../../../playground/frontend/playground_components
 flutter pub get
-flutter pub run build_runner build
```

Comment:
Will this work instead of these 3 lines?
```bash
./gradlew :playground:frontend:generateCode
```

Reply:
Or rather
```bash
./gradlew :playground:frontend:playground_components:generateCode
```

Reply:
@alexeyinkin and I have updated readme.